Makefile

```diff
@@ -309,7 +309,7 @@ submodulesclean:
 	@git submodule update --init --recursive --force
 
 distclean: submodulesclean
-	@git clean -ff -x -d
+	@git clean -ff -x -d -e ".project" -e ".cproject"
```

Comment:
Yes, that makes sense. I've lost my project settings so many times I've learned to be happy with the defaults...